programs/programs/wa/orca_lift/tests/test_wa_orca_lift.py

```
@@ -0,0 +1,223 @@
+        e = self._run_household(calc)
+        self.assertTrue(e.eligible)
+
+    def test_wic_bypasses_income(self):
+        calc = make_calculator(has_wic=True, yearly_income=60_000, fpl_annual=15_960)
+        e = self._run_household(calc)
+        self.assertTrue(e.eligible)
+
+    # --- Age gate ---
+
+    def test_no_member_19_64_ineligible_even_with_snap(self):
+        # Household has SNAP and low income but no one aged 19–64 → ineligible
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)